src/Model/Worker.py

```diff
@@ -53,6 +54,8 @@ def __init__(self, fn, *args, **kwargs):
 
         if 'progress_callback' in self.kwargs and self.kwargs['progress_callback'] is True:
```

Comment:
blah is True
can more efficiently be left as
blah
`if 'progress_callback' in self.kwargs and self.kwargs['progress_callback']:`
```suggestion
        if 'progress_callback' in self.kwargs and self.kwargs['progress_callback']:
```

Reply:
no need...
this is why we have code review, and this in particular was a nit